packages/@react-spectrum/s2/src/Meter.tsx

```diff
@@ -40,6 +42,8 @@ export interface MeterProps extends Omit<AriaMeterProps, 'children' | 'className
   label?: ReactNode
 }
 
+export const MeterContext = createContext<ContextValue<MeterProps, DOMRefValue<HTMLDivElement>>>(null);
```

Comment:
Do we want to prefix all our new contexts with "Spectrum"? so "SpectrumMeterContext"?

Just to reduce confusion on names since now there is RAC's MeterContext and Spectrum's MeterContext

I know I'd have a tough time if I was working in another project. Yes, I could rename the import like we're doing in this PR to "AriaMeterContext", but then the convention may change project to project. If we started by naming them separately then at least most would go with the default naming scheme.

Reply:
🤷 open question for the team to discuss. I don't think renaming them is too hard.

Reply:
I'm ok with either. On one hand we already have duplicate naming between RAC and S2 so that would be consistent, and on the other hand it would be easier to read the codebase with the prefix.

Reply:
The prefix seems like it help people always find the right file or code and cause less confusion.